source/features/clone-branch.tsx

```diff
@@ -4,21 +4,27 @@ import select from 'select-dom';
 import delegate, {DelegateEvent} from 'delegate-it';
 import * as api from '../libs/api';
 import features from '../libs/features';
-import {getCleanPathname} from '../libs/utils';
+import {getRepoURL} from '../libs/utils';
 
 async function cloneBranch(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): Promise<void|false> {
 	const currentTarget = event.delegateTarget;
 	const branchElement = (currentTarget.closest('[data-branch-name]') as HTMLAnchorElement);
-	const newBranchName = prompt('Enter the new branch name');
+	const newBranchName = prompt('Enter the new branch name')?.trim();
```

Comment:
the `trim` is for just in case someone adds an extra space at the end

Reply:
Doesn’t the API already handle all unsupported characters?

Reply:
Should I add a coment?